scripts/data/cms/index.ts

```diff
@@ -0,0 +1,87 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+import got from "got";
+const { DATO_API_TOKEN } = process.env;
+const mockDataEnabled = process.env.USE_MOCK_DATA_CMS === "1";
+
+async function getFromDato(query: string, rootProp: string) {
+  const response = await got("https://graphql.datocms.com/", {
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${DATO_API_TOKEN}`,
+    },
+    json: true,
+    body: {
+      query,
+      variables: null,
+    },
+  });
+  return response.body.data[rootProp];
+}
```

Comment:
Thoughts on using the native Node.js `fetch` function here? It got added in v16.15.0 so should be available. Means we can drop `got` and the got types package.